modules/web/client/src/main/scala/observe/ui/components/sequence/steps/ObservationProgressBar.scala

```diff
@@ -55,13 +55,14 @@ object ObservationProgressBar extends ProgressLabel:
           // TODO Smooth Progress Bar
           // val remainingMillis = p.maxValue - s.value
 
-          val totalMillis     = total.toMilliseconds.toInt
+          // val totalMillis     = total.toMilliseconds.toInt
```

Comment:
Commented code?

Reply:
This will revert once we have progress signals from the server.

Reply:
This will revert once we have progress signals from the server.